src/EngineManager.php

```diff
@@ -59,6 +65,31 @@ protected function ensureAlgoliaClientIsInstalled()
         throw new Exception('Please install the Algolia client: algolia/algoliasearch-client-php.');
     }
 
+    /**
+     * Set the default Algolia config headers.
+     *
+     * @return array
+     */
+    protected function defaultAlgoliaHeaders()
+    {
+        if (! config('scout.user')) {
+            return [];
+        }
+
+        $headers = [];
+        $request = $this->container->make(Request::class);
```

Comment:
Because we are using `config()` on the 75, maybe we can use `request()` here too?